index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="styles/main.css">
     <link rel="stylesheet" href="styles/mobile.css">
     <link rel="stylesheet" href="styles/baguetteBox.css">
-    <link rel="stylesheet" href="styles/baguetteBox.js">
+    <script src="styles/baguetteBox.js"></script>
```

Comment:
Don't include javascript files using the `<link>` element.  Use `<script>` instead